docs/tutorial/force_calibration/calibration_items.rst

```diff
@@ -62,19 +69,23 @@ These parameters are properties and can be extracted as such::
 Redoing a Bluelake calibration
 ------------------------------
 
-In order to redo the Bluelake calibration, the force data that was used for calibration has to be included in the h5 file.
-Note that this force data is not exported nor marked by default; it has to be explicitly added to the exported file.
+.. important::
+    In order to redo a Bluelake calibration, the force data that was used for the calibration has to
+    be included in the `.h5` file. *Note that this force data is not exported nor marked by default*;
+    it has to be explicitly added to the exported file.
+
 We can directly slice the channel by the calibration item we want to reproduce to extract the relevant data::
 
     force1x_slice = f.force1x[f.force1x.calibration[1]]
 
-To recalibrate data we first have to de-calibrate the data to get back to voltages.
+To recalibrate data we first have to de-calibrate the data to get back to raw voltage.
 To do this, we divide our data by the force sensitivity that was active at the start of the slice.
 
     >>> old_calibration = force1x_slice.calibration[0]
     ... volts1x_slice = force1x_slice / old_calibration.force_sensitivity
 
-To reproduce a calibration performed with Bluelake, the easiest way to extract all the relevant parameters is to use :meth:`~lumicks.pylake.force_calibration.calibration_item.ForceCalibrationItem.calibration_params()`::
+The easiest way to extract all of the relevant calibration parameters is to use
```

Comment:
This is not entirely correct. This API provides the parameters you should provide Pylake with the redo the calibration. They are not the output parameters, but the input ones.

That said, the fact that you changed it makes me realize it wasn't clear, so I rephrased it a little based on your suggestion.

```rst
The easiest way to extract all the relevant input parameters for a calibration is to use
:meth:`~lumicks.pylake.force_calibration.calibration_item.ForceCalibrationItem.calibration_params()`::
```